PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Pull Request Requirements
+
+Pull Requests must be rebased onto master and squashed into a single commit to
```

Comment:
@ruzell22 Please use `main` instead as the default branch name and also make sure to mention that the upstream main branch is the one to rebase onto (beginners will assume that their own main branch on their for is the one which ends up being outdated as well)

Reply:
Hello @petermetz , I have changed the master into upstream/main and also changed the paragraphs into bulleted type to act as checklist. The links were also changed as well so it will not flood the template

Reply:
@ruzell22 Thank you!